### DISCUSSION TITLE
Increase Time Precision

### DISCUSSION THREAD

**Discussion by f870103** [redraskal/r6-dissect#64] [Ideas]
The output precision is second, is it possible to set it to millisecond?

This will help a lot when tracking defuser-related events (e.g. defender team successfully counter-defused with less than one second to round end).

Also, is it possible to add events like `DefuserPlantStop` and `DefuserDisableStop`?

**Comment by redraskal** [comment]
@f870103 this should be possible. output precision should not be difficult to change, but i have not yet found how to identify plant/disable cancellations
